.circleci/config.yml

```diff
@@ -30,6 +30,7 @@
               source activate jiant
               python -m nltk.downloader perluniprops  nonbreaking_prefixes punkt
               python main.py --config_file config/demo.conf
+              python main.py --config_file config/demo.conf --overrides "do_pretrain = 0, do_target_task_training = 0, load_model = 1"
```

Comment:
Don't we need to set load_target_train_checkpoint (or with the new change in the Fixing Target Train PR) load_eval_checkpoint here?

Reply:
Ah, yeah. This is working as is, but producing at-chance results. 

I think that's actually a bug: If there are checkpoints for saved target task models, we should load them when starting evaluation. That was the historic behavior, and I think it's the most user-friendly. Can we try to address that in the final version of the checkpointing rewrite PR?

Reply:
Ah got it - so we should automatically load the best target train without having to specify it. 
I think this is addressed in the checkpointing rewrite PR right now.